spec/components/schemas/reports/message/entry/base.ts

```diff
@@ -0,0 +1,43 @@
+// tslint:disable:max-line-length
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../../utils/ref';
+
+const base: SchemaObject = {
+  title: 'Entry object for Message Report',
+  description: 'This is a entry object model for Message Report.',
+  type: 'object',
+  properties: {
+    channel: {
+      title: 'Channel',
+      type: 'string',
+      readOnly: true,
+    },
+    type: {
+      title: 'Type of message',
+      description: 'Type of message, that may be either "message" or "notification"',
+      type: 'string',
+      readOnly: true,
+    },
+    directionInTotal: {
+      title: 'Total of "IN" messages',
+      description: 'Counter to sum messages of IN direction.',
```

Comment:
Aqui acho que fica melhor 'The sum of the messages of the IN direction.'

E, de novo, não lembro como está no resto, mas deveria ter consistência: se tem aspas no title deveria ter no description. Então, ou tira do title, ou põe na description.

Reply:
O mesmo que falei do out abaixo. Counter não faz sentido pois isso não é uma variável em um código e sim um resultado de query.